translations/base-fr.yaml

```diff
@@ -190,7 +193,9 @@ dialogs:
     confirmSavegameDelete:
         title: Confirmez la suppression
         text: >-
-            Êtes-vous sûr de vouloir supprimer votre partie ?
+            Êtes-vous sûr de vouloir supprimer cette partie ?<br><br>
+            '<savegameName>' au niveau <savegameLevel><br><br>
+            Cela sera définitif!
```

Comment:
Add ` ` (U+202F NARROW NO-BREAK SPACE) before `!`

Reply:
I agree with you for no-break-space but I have modified the file inside the web browser. I do not know how to have this char.
Have you an idea?